### PR TITLE
Getting passing builds that fix bugs.

### PR DESCRIPTION
**Pull request by fermadeiral**
Getting passing builds that fix bugs by checking whether their previous builds are failing builds with failing tests. The state of these passing builds is `ProjectState.FIXERBUILD`.

Solver path used by Nopol is not required for FORBEARS mode anymore.

### CODE REVIEW

File: repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java [hunk 1]
```diff
@@ -243,7 +249,9 @@ private void run(String[] args) throws JSAPException, IOException {
             System.exit(-1);
         }
         this.checkToolsLoaded();
-        this.checkNopolSolverPath();
+        if (RepairMode.valueOf(arguments.getString("mode").toUpperCase()) != RepairMode.FORBEARS) {
```
Comment (surli) [inline repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java]: We can initialize the field mode right now and then you can use it in your condition

**Comment by surli** [comment]
We should discuss about the design of this PR: you introduce creation of inspectors inside a step, so it introduces a new complexity I'd like to avoid.

File: repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java (hunk 1 above)
Comment (fermadeiral) [inline repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java]: Ok.

**Comment by fermadeiral** [comment]
I'll try something and I'll send you. If the issue remains, then we discuss.

**Pull request closed without merging**